datasets/lama/README.md

```diff
@@ -0,0 +1,314 @@
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:**
+https://github.com/facebookresearch/LAMA
+- **Repository:**
+https://github.com/facebookresearch/LAMA
+- **Paper:**
+@inproceedings{petroni2019language,
+  title={Language Models as Knowledge Bases?},
+  author={F. Petroni, T. Rockt{\"{a}}schel, A. H. Miller, P. Lewis, A. Bakhtin, Y. Wu and S. Riedel},
+  booktitle={In: Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing (EMNLP), 2019},
+  year={2019}
+}
+
+@inproceedings{petroni2020how,
+  title={How Context Affects Language Models' Factual Predictions},
+  author={Fabio Petroni and Patrick Lewis and Aleksandra Piktus and Tim Rockt{\"a}schel and Yuxiang Wu and Alexander H. Miller and Sebastian Riedel},
+  booktitle={Automated Knowledge Base Construction},
+  year={2020},
+  url={https://openreview.net/forum?id=025X0zPfn}
+}
+
+### Dataset Summary
+
+This dataset provides the data for LAMA. The dataset include a subset
+of Google_RE
+(https://code.google.com/archive/p/relation-extraction-corpus/), TRex
+(subset of wikidata triples), Conceptnet
+(https://github.com/commonsense/conceptnet5/wiki) and Squad. There are
+configs for each of "google_re", "trex", "conceptnet" and "squad",
+respectively.
+
+The dataset includes some cleanup, and addition of a masked sentence
+and assocaited answers for the [MASK] token. The accuracy in
+predicting the [MASK] token shows how well the language model knows
+facts and common sense information. The [MASK] tokens are only for the
+"object" slots.
+
+This version of the dataset includes "negated" sentences as well as
+the masked sentence. Also, certain of the config includes "template"
+and "template_negated" fields of the form "[X] some text [Y]", where
+[X] and [Y] are the subject and object slots respectively of certain
+relations.
+
+See the paper for more details.  For more information, also see:
+https://github.com/facebookresearch/LAMA
+
+### Languages
+en
+
+## Dataset Structure
+
+### Data Instances
+
+
+The trex config has the following fields:
+
+
+``
+{
+	'uuid': ...,
+	'obj_uri': ...,
+	'obj_label': ...,
+	'sub_uri': ...,
+	'sub_label': ...,
+	'predicate_id': ...,
+	'sub_surface': ...
+	'obj_surface': ...
+	'masked_sentence': ...
+	'template': ...
+	'template_negated': ...
+	'label': ...
+	'description': ...
+	'type': ...
+}

```

Comment:
Here we expect actual samples from the dataset.
You can get one by using `load_dataset` and taking the first sample `dataset["train"][0]` for example.

If the sample is too long in you opinion feel free to truncate it and mention that it was truncated

Reply:
done